app/src/main/res/values/colors.xml

```diff
@@ -1,12 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="tusky_blue">#2b90d9</color>
+    <color name="tusky_blue_darker">#217aba</color>
     <color name="tusky_blue_light">#56a7e1</color>
     <color name="tusky_orange">#ca8f04</color>
     <color name="tusky_orange_light">#fab207</color>
-    <color name="tusky_green">#19a341</color>
+    <color name="tusky_green">#00731B</color>
```

Comment:
The red and green changes (and mostly move them to named color values) concern the diff display.

How did you select those in the first place? @nikclayton 

For example I now used the existing red from colors.xml (came from #1952) which is df1553 not your clean red df0000.
But for the remainder I used your values.

the existing Green is only used for 3 spark buttons (are they used?). Red is used in a few more places.

Reply:
The existing green/red colours didn't work on a dark background, see https://github.com/tuskyapp/Tusky/pull/3314#pullrequestreview-1320455937

Reply:
Just saw I missed your query about spark buttons.

They are used -- they're the buttons that provide the animations when you reply, boost, or bookmark a status.